index.html

```diff
@@ -1,12 +1,86 @@
-  </head>
-  <body>
 
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/htmlcss">HTML и CSS, уровень 1</a>».</p>
+<head>
+  <meta charset="UTF-8">
+  <title>Sedona</title>
+</head>
+
+<body>
+  <header>
+    <section>
+      <h2>Навигация</h2>
+      <nav>
+        <a href="index.html"> <img src="https://via.placeholder.com/138x70" alt="Логотип" width="138" height="70"> </a>
+        <ul>
+          <li><a href="#">Информация</a></li>
+          <li><a href="#">Фото и видео</a></li>
+          <li><a href="#">Карта штата</a></li>
+          <li><a href="catalog.html">Гостиницы</a></li>
+        </ul>
+      </nav>
+    </section>
+  </header>
+  <main>
+    <section>
+      <h1>Седона — небольшой городок в Аризоне, заслуживающий большего! </h1>
+      <p>Рассмотрим 5 причин, по которым Седона круче, чем гранд каньон!</p>
+      <ul>
+        <li>
+          <h3>Настоящий городок</h3>
+          <p>— №1 —</p>
+          <p>Седона - не аттракцион для туристов, там течёт своя жизнь</p> <img src="https://via.placeholder.com/800x256" alt="Жилье" width="800" height="256">
+          <ul>
+            <li>
+              <h3>Жилье</h3>
+              <p>Рекомендуем пожить в настоящем мотеле, всё как в кино!</p>
+            </li>
+            <li>
+              <h3>Еда</h3>
+              <p>Всегда заказывайте фирменный бургер, вы не разочаруетесь!</p>
+            </li>
+            <li>
+              <h3>Cувениры</h3>
+              <p>Не только китайского, но и местного производства!</p>
+            </li>
+          </ul>
+          <li> <img src="https://via.placeholder.com/800x256" alt="Жилье" width="800" height="256">
+            <h3>Там есть Мост дьявола</h3>
+            <p>— №2 —</p>
+            <p>Да, по нему можно пройти! Если конечно вы осмелитесь</p>
+            <li>
+              <h3>Небольшая площадь</h3>
+              <p>— №3 —</p>
+              <p>все достопримечательности находятся очень близко</p>
+            </li>
+            <li>
+              <h3>красивая дорога</h3>
+              <p>— №4 —</p>
+              <p>ехать в седону из лас-вегаса совсем не скучно!</p>
+            </li>
+            <li>
+              <h3>мало туристов</h3>
+              <p>— №5 —</p>
+              <p>большинство едет в гранд каньон и толпится там</p>
+            </li>
+      </ul>
+    </section>
+    <section>
+      <h2>Заинтересовались?</h2>
+      <p>Укажите предполагаемые даты поездки, и мы покажем вам лучшие предложения гостиниц в седоне</p>
+    </section>
+  </main>
+  <footer>
+    <div>
+      <p>#visitSEDONA</p> <a href="tel:+74956660166">ТЕЛ: +7(495) 666-01-66</a> </div>
+    <div>
+      <ul>
+        <li><a href="#">Twitter</a></li>
+        <li><a href="#">Facebook</a></li>
+        <li><a href="#">Youtube</a></li>
+      </ul>
+      <p>Website by</p> <a href="https://up.htmlacademy.ru/htmlcss/30/project/sedona#sedona-3-8">htmlacademy</a> </div>
```

Comment:
переноси теги на новые строки
```suggestion
      <p>Website by</p>
      <a href="здесь вставить правильную ссылку">htmlacademy</a>
    </div>
```